### ISSUE TITLE
Add "accessibility-ready" to tag list

### ISSUE THREAD

**Issue by nielslange** [WordPress/twentytwenty#774]
See discussion in https://wordpress.slack.com/archives/C02RP4VMP/p1570318331370500 and https://wordpress.slack.com/archives/C02RP4VMP/p1570318331370500

**Comment by ntwb** [comment]
Before merging #774 adding the `accessibility-ready` tag, is the theme actually "accessibility-ready” ready per the Theme Review Teams requirements?

If it is, then the PR can be merged, if not the theme should be updated to make it so.

**Comment by nielslange** [comment]
@ntwb That sound like a good plan. 

@carolinan & @afercia Do you have a vague idea of how many issues still need to be addressed to get Twenty Twenty `accessibility-ready` according to https://make.wordpress.org/themes/handbook/review/accessibility/?

**Comment by ntwb** [comment]
Closing this in favour of #785
